connect/eaas/manager.py

```diff
@@ -222,13 +222,13 @@ async def build_bg_response(self, task_data, future):
         """
         Wait for a background task to be completed and than uild the task result message.
         """
-        result_message = TaskPayload(**task_data.to_json())
+        result_message = TaskPayload(**task_data.to_dict())
```

Comment:
Not sure, but looks like for copying the instances there is a specific method - [link](https://docs.python.org/3/library/dataclasses.html#dataclasses.replace)